packages/docs/README.md

```diff
@@ -72,3 +74,17 @@ This tab will not show up when the site is built in production mode.
 
 It is a good idea to check the PreTeXt log info in the _Debug Info_ tab to make sure you're not creating an example with
 invalid PreTeXt.
+
+### Adding element and attribute file templates
+
+When the schema is updated, to get new template files for new elements or attributes, you can run a script to add just missing pages.  This can be done with `tsx`, from the root of the `pacckages/docs` folder:
+
+```
+npx tsx scripts/create-element-templates.ts
```

Comment:
Probably better to use `npx vite-node scripts/create-element-templates.ts` Should be more robust (and will also ignore typescript errors :-D).